website/docs/r/policy_ip_block_quota.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: "Multitenancy"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_ip_block_quota"
+description: A resource to configure IP Block on NSX Policy.
+---
+
+# nsxt_policy_ip_block_quota
+
+This resource provides a means to configure IP Block Quota on NSX Policy.
+
+This resource is applicable to NSX Policy Manager.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_ip_block_quota" "small" {
+  display_name = "small-quota"
+  description  = "terraform provisioned quota"
+
+  quota {
+    ip_block_paths        = [nsxt_policy_ip_block.all.path]
+    ip_block_visibility   = "EXTERNAL"
+    ip_block_address_type = "IPV4"
+
+    single_ip_cidrs = 4
+    other_cidrs {
+      total_count = 4
+    }
+  }
+}
+```
+
+## Example Usage - Multi-Tenancy
+
+```hcl
+resource "nsxt_policy_ip_block_quota" "small" {
+  context {
+    project_id = "dev"
+  }
+
+  display_name = "small-quota"
+  description  = "terraform provisioned quota"
+
+  quota {
+    ip_block_paths = [nsxt_policy_ip_block.dev.path]
+
+    single_ip_cidrs = 4
+    other_cidrs {
+      total_count = 4
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `context` - (Optional) The context which the object belongs to
+    * `project_id` - (Required) The ID of the project which the object belongs to
+* `display_name` - (Required) The display name for the IP Block.
+* `description` - (Optional) Description of the resource.
+* `quota` - (Required) Quota specification
+  * `ip_block_paths` - (Required) List of IP blocks that this quota applies to.
+  * `ip_block_address_type` - (Optional) One of `IPV4`, `IPV6`. A quota will be applied on block of same address type. One v4 block and another v6 block cannot be speficied within the same quota.
+  * `ip_block_visibility` - (Optional) One of `EXTERNAL`, `PRIVATE`. A quota will be applied on blocks with same visibility. Private and External blocks cannot be specified within the same block.
+  * `single_ip_cidrs` - (Optional) Quota for single IP CIDRs allowed. Default is -1 (unlimited). 
+  * `other_cidrs` - (Required) Quota for other cidrs
+    * `mask` - (Optional) Largest mask size that is allowed
```

Comment:
I think we need to specify how this is specified (e.g.: /28).
User might think this is a subnet size specification (integer)